internal/driver/driver.go

```diff
@@ -73,6 +73,9 @@ func generateRawReport(p *profile.Profile, cmd []string, cfg config, o *plugin.O
 
 	cfg = applyCommandOverrides(cmd[0], c.format, cfg)
 
+	// Create label pseudo-nodes before filtering, in case the filters use the generated nodes.
```

Comment:
nit: please keep comments in 80 columns.

Reply:
Done